webhooks-extension/src/components/WebhookCreate/WebhookCreate.scss

```diff
@@ -0,0 +1,185 @@
+.webhook-create {
+
+  .bx--inline-notification {
+    width: 100%;
+    max-width: 100%;
+    min-width: 100%;
+  }
+
+}
+
+.create-container {
+  padding-top: 3rem;
+  background-color: white;
+  
+  .title {
+    font-size: 150%;
+    text-align: center;
+    padding-bottom: 3rem;
+  }
+  
+  .row {  
+    display: flex;
+    flex-wrap: nowrap;
+    align-items: center;
+    padding: 1% 0% 1% 15%;
+  }
+  
+  .item-label {
+    width: 15%;
+    background-color: white;
+    text-align: left;
+  }
+  
+  .help-icon {
+    padding-right: 1%;
+    width: 4%;
+  }
+  
+  .entry-field {
+    background-color: white;
+    width: 60%;
+  }
+  
+  .git-access-drop-down-div {
+    background-color: white;
+    width: 50%;
+  }
+  
+  #secButtonEnabled {
+    cursor: pointer;
+  }
+  
+  #secButtonDisabled {
+    cursor: unset;
+    fill: #dcdcdc;
+    pointer-events: none;
+  }
+  
+  #pipeline, #namespace, #git, #sa {
+    margin: 0;
+  }
+  
+  .add-sec-btn {
+    width: 5%;
+    display: flex;
+    align-items: center;
+    fill: green;
+    justify-content: center;
+  }
+  
+  .del-sec-btn {
+    width: 5%;
+    display: flex;
+    align-items: center;
+    fill: red;
+    justify-content: center;
+  }
+  
+  .token {
+    align-items: center;
+    display: flex;
+  }
+  
```

Comment:
Might be able to reduce some duplication of styles here, for example the only difference between these two is the visibility

Reply:
@extend added to numerous elements